### PR TITLE
Mixers monitor + build option

### PR DESCRIPTION
**Pull request by davxlw**

### CODE REVIEW

File: companion/src/firmwares/opentx/opentxTaranisSimulator.cpp [hunk 1]
```diff
@@ -57,6 +57,7 @@
 #define HAPTIC
 #define REVPLUS
 #define OVERRIDE_CHANNEL_FUNCTION
+#define MIXERS_MONITOR
```
Comment (bsongis) [inline companion/src/firmwares/opentx/opentxTaranisSimulator.cpp]: No!!!

File: companion/src/firmwares/opentx/opentxTaranisX9ESimulator.cpp [hunk 2]
```diff
@@ -59,6 +59,7 @@
 #define REVPLUS
 #define REV9E
 #define OVERRIDE_CHANNEL_FUNCTION
+#define MIXERS_MONITOR
```
Comment (bsongis) [inline companion/src/firmwares/opentx/opentxTaranisX9ESimulator.cpp]: No!!!

File: companion/src/firmwares/opentx/opentxinterface.cpp [hunk 3]
```diff
@@ -1290,6 +1292,7 @@ void registerOpenTxFirmwares()
   openTx->addOption("haptic", QObject::tr("Haptic module installed"));
   openTx->addOption("lua", QObject::tr("Support for Lua model scripts"));
   openTx->addOption("nojoystick", QObject::tr("No Joystick emulation inside the FW (only Mass Storage as in the Bootloader)"));
+  openTx->addOption("mixersmon", QObject::tr("Adds mixers output view to the CHANNELS MONITOR screen, pressing [ENT] switches between the views"));
```
Comment (bsongis) [inline companion/src/firmwares/opentx/opentxinterface.cpp]: Ok. I just need to think to update the compilation server which needs to support it.

**Comment by bsongis** [comment]
Otherwise it's ok!

**Comment by davxlw** [comment]
I removed the simu mods.

**Comment by bsongis** [comment]
Thanks!

**Pull request merged**